core/src/main/java/org/apache/shiro/mgt/SessionsSecurityManager.java

```diff
@@ -149,6 +149,9 @@ protected void applyEventBusToSessionManager() {
     }
 
     public Session start(SessionContext context) throws AuthorizationException {
+        if (sessionManager == null) {
+            throw new IllegalStateException("Session manager is destroyed");
```

Comment:
```suggestion
            throw new IllegalStateException("Session manager is not available or has been destroyed");
```
It's possible the session manager could have been set to null for some other reason (though it looks like that would take a child class overriding a few different methods). If you think this suggestion doesn't add any clarity please reject it.

Reply:
Ok, thanks, I added the suggestion.